mgmt/rest/v1/plugin.go

```diff
@@ -74,6 +74,9 @@ func (s *apiV1) loadPlugin(w http.ResponseWriter, r *http.Request, _ httprouter.
 		return
 	}
 	if strings.HasPrefix(mediaType, "multipart/") {
+		var certPath string
+		var keyPath string
+		var caCertPaths string
```

Comment:
It should be, but the documentation requires refresh before any new information may be added. See issue #1629.